app/src/androidTest/java/com/github/h3lp3rs/h3lp/MySkillsActivityTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.h3lp3rs.h3lp
+
+import android.app.Activity
+import android.app.Instrumentation
+import android.content.Context
+import android.content.Intent
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.matcher.IntentMatchers
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.Matchers
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MySkillsActivityTest {
+    private val ctx: Context = ApplicationProvider.getApplicationContext()
+
+    @get:Rule
+    val testRule = ActivityScenarioRule(
+        MySkillsActivity::class.java
+    )
+
```

Comment:
Just a detail, here you can import Intents and Espresso for example to avoid calling them each time

Reply:
Yep, same thing for Espresso